### PR TITLE
add camera client kivy example

### PR DESCRIPTION
**Pull request by edgarriba**
image

image

### CODE REVIEW

File: brain/examples/kivy_viz/main.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        self.tasks += [asyncio.ensure_future(self.stream_camera(self.client))]
+
+        return await asyncio.gather(run_wrapper(), *self.tasks)
+
+    async def stream_camera(self, client: OakCameraClient) -> None:
+        while self.root is None:
+            await asyncio.sleep(0.01)
+
+        # get the streaming object
+        response_stream = client.stream_frames(every_n=self.stream_every_n)
```
Comment (ethanrublee) [inline brain/examples/kivy_viz/main.py:81]: Does this need an await?

File: brain/examples/kivy_viz/main.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+        # start the streaming service
+        await client.start_service()
+
+        while True:
+            # query the service state
+            state: oak_pb2.OakServiceState = await client.get_state()
+
+            if state.value != oak_pb2.OakServiceState.RUNNING:
+                print("Camera is not streaming!")
```
Comment (ethanrublee) [inline brain/examples/kivy_viz/main.py:91]: Perhaps this should sleep to not spam the service with get_state requests.

File: brain/examples/kivy_viz/main.py (hunk 1 above)
Comment (edgarriba) [reply brain/examples/kivy_viz/main.py:81]: just to get the object not needed i believe, we place it in the read call

File: brain/examples/kivy_viz/main.py (hunk 2 above)
Comment (edgarriba) [reply brain/examples/kivy_viz/main.py:91]: sure

**Pull request merged**